<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleOpenCVExampleControls</class>
 <widget class="QWidget" name="SimpleOpenCVExampleControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>287</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="m_InputID"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="m_SeparateWindow">
     <property name="text">
      <string>Render in Separate Window</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_MITKImage">
     <property name="text">
      <string>Render as MITK Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_StartGrabbing">
     <property name="text">
      <string>Start Video Grabbing</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
